Integrations/VulnDB/VulnDB.yml

```diff
@@ -141,13 +141,15 @@ script:
       secret: false
     - default: false
       defaultValue: '20'
-      description: Maximum number of entries to return. A high number of entries might effect performance.
+      description: Maximum number of entries to return. A high number of entries might
+        effect performance.
       isArray: false
       name: max_size
       required: false
       secret: false
     deprecated: false
-    description: Returns all, or specific, vendor details to include vendor name and ID.
+    description: Returns all, or specific, vendor details to include vendor name and
+      ID.
```

Comment:
This description isn't so clear. If you could try and make it better that would be great, but either way, a technical writer will review so if you want to wait for their review, that's also ok.

Reply:
This is actually not my doing and I'm not sure what this API does, it must have been altered by accident.

As for now I don't really know what this API should do, do you want me to dig deeper into it?

In the meantime- I returned it to the original